src/main/resources/assets/extratrees/lang/en_US.lang

```diff
@@ -58,8 +58,10 @@ extratrees.block.planks.olive=Olive
 extratrees.block.planks.sweetgum=Sweetgum
 extratrees.block.planks.rosewood=Rosewood
 extratrees.block.planks.gingko=Gingko
-extratrees.block.planks.pinkivory=Pink Ivory
-
+extratrees.block.planks.pink ivory=Pink Ivory
```

Comment:
This is wrong key. Need without space.
See screenshot below.
https://cloud.githubusercontent.com/assets/1331559/26763298/558b19f2-4959-11e7-9ef2-6889bc20705c.png